german/the-black-hack.md

```diff
@@ -113,7 +113,7 @@ Zeit im Spiel wird vor allem auf zwei Arten nachgehalten - **Momente** (Runden)
 
 ## Bewegung und Entfernung
 
-Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Benachbart**, **in der Nähe**, **abseits** und **entfernt**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
+Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Dicht**, **Nah**, **Weit** und **Fern**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
```

Comment:
Auch wenn es wenige umgangssprachlich ist, würde ich für Version 1 stimmen. Gerade in Sätzen wie "Ich bewege mich In Die Nähe des Orks" funktioniert In Der Nähe (oder auch In Der Ferne) nicht mehr. Hier würde ich die Eindeutigkeit am Spieltisch der Umgangssprache vorziehen.